roles/st2/tasks/main.yml

```diff
@@ -1,17 +1,25 @@
 ---
-- name: Install latest st2 package
+- name: Install libffi-devel on EL6
   become: yes
-  apt:
+  package:
+    name: ftp://fr2.rpmfind.net/linux/centos/6/os/x86_64/Packages/libffi-devel-3.0.5-3.2.el6.x86_64.rpm
+    state: latest
+  when: ansible_os_family == "RedHat" and ansible_distribution_major_version == "6"
+  ignore_errors: yes  # Ignore error when already installed
+
+- name: Install latest st2 package (stable)
```

Comment:
`(stable)` is not exactly what we install here.

Package repository (if you meant that) is controlled via `st2_pkg_repo`, see: https://github.com/StackStorm/ansible-st2#variables

So `(stable)` note could be removed